### PR TITLE
Missed a edits

### PR DESCRIPTION
**Pull request by paul-grassi**
Sorry, caught some unassigned for -3 and wrapped them up.

### CODE REVIEW

File: sp800-63-3/sec1_2_introduction.md [hunk 1]
```diff
@@ -80,8 +80,7 @@ Privacy Provisions of the E-Government Act of 2002* [[M-03-22]](#M-03-22). See t
 Transactions* [[ESIG]](#ESIG) for additional information on legal risks, especially those that are related to the need to satisfy legal standards of proof and prevent repudiation.
 
 Additionally, Federal agencies implementing these guidelines should
-adhere to the requirements of Title III of the E-Government Act,
-entitled the *Federal Information Security Modernization Act* [[FISMA]](#FISMA), and related NIST standards and guidelines. FISMA directs Federal agencies to develop, document, and implement
+adhere to the requirements of the *Federal Information Security Modernization Act* [[FISMA]](#FISMA), and related NIST standards and guidelines. FISMA directs Federal agencies to develop, document, and implement
```
Comment (jimfenton) [inline sp800-63-3/sec1_2_introduction.md:83]: Title III is also referenced in the definition of FISMA in the Definitions appendix.

File: sp800-63-3/sec5_DIRM.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ As such, agencies SHALL assess the risk of proofing, authentication, and federat
...
+Risk assessments determine the extent to which risk must be mitigated by the identity proofing, authentication, and federation processes. These determinations drive the relevant choices of applicable technologies and mitigation strategies, rather than the desire for any given technology driving risk determinations. Once an agency has completed the overall risk assessment; selected individual assurance levels for identity proofing, authentication, and federation (if applicable); and determined the processes and technologies they will will employ to meet each assurance level, agencies SHALL develop a "Digital Identity Acceptance Statement", in accordance with NIST 800-53r5 IA-1 a.1. See [Section 5.5.](#daps) for more detail on the necessary content of the Digital Identity Acceptance Statement.
```
Comment (jimfenton) [inline sp800-63-3/sec5_DIRM.md]: Should include citation to SP 800-53.
nit: callout to section number doesn't have a trailing period: "Section 5.5." -> "Section 5.5"

File: sp800-63-3/sec4_model.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ A digital authentication system may incorporate multiple factors in one of two w
...
 
-For example, consider a piece of hardware (the authenticator) that contains a cryptographic key (the authenticator secret) where access is protected with a fingerprint. When used with the biometric, the cryptographic key produces an output that is used in the authentication process to authenticate the claimant. An impostor must steal the encrypted key (by stealing the hardware) and replicate the fingerprint to use the authenticator. These guidelines consider such a device effective to provide MFA, although the actual authentication protocol between the verifier and the claimant simply proves possession of the key.
+For example, item 1 can be satisfied by pairing a memorized secret (what you know) with an out-of-band-device (what you have). Both authenticator outputs are presented to the verifier to authenticate the claimant. For item 2, consider a piece of hardware (the authenticator) that contains a cryptographic key (the authenticator secret) where access is protected with a fingerprint. When used with the biometric, the cryptographic key produces an output that is used in the authentication process to authenticate the claimant.
```
Comment (jimfenton) [inline sp800-63-3/sec4_model.md]: nit: "out-of-band-device" -> "out-of-band device"

Possibly remove "in the authentication process" in last sentence. Seems redundant.

**Review by jimfenton — CHANGES_REQUESTED** [review]
Very minor changes needed.

File: sp800-63-3/sec1_2_introduction.md (hunk 1 above)
Comment (paul-grassi) [reply sp800-63-3/sec1_2_introduction.md:83]: I'm not seeing it in the current version of definitions.md

File: sp800-63-3/sec5_DIRM.md (hunk 2 above)
Comment (paul-grassi) [reply sp800-63-3/sec5_DIRM.md]: 5.5. has the trailing period but i added reference to 53

File: sp800-63-3/sec1_2_introduction.md (hunk 1 above)
Comment (jimfenton) [reply sp800-63-3/sec1_2_introduction.md:83]: My mistake; I was looking at nist-pages, not current version.

**Review by jimfenton — APPROVED** [review]

**Pull request merged**